src/components/Search/SearchStatusBar.tsx

```diff
@@ -177,10 +177,16 @@ function SearchStatusBar({queryJSON, onStatusChange}: SearchStatusBarProps) {
             {options.map((item, index) => {
                 const onPress = singleExecution(() => {
                     onStatusChange?.();
-                    const query = SearchQueryUtils.buildSearchQueryString({...queryJSON, status: item.status});
+                    let newStatus;
+                    if (Array.isArray(queryJSON.status)) {
+                        newStatus = queryJSON.status.includes(item.status) ? queryJSON.status.filter((status) => status !== item.status) : [...queryJSON.status, item.status];
```

Comment:
I think we should only allow selecting multiple statuses with maybe a keyboard shortcut or long press on mobile. We should also not allow deselecting all of them, which is currently possible.

Reply:
In the meantime, I've reverted my changes and only one status can be selected.

See https://github.com/Expensify/App/pull/52185#issuecomment-2464246557